templates/frontEndLayout.html

```
@@ -289,16 +299,218 @@
+            imageSmall: "static/alfalfa.jpg",
+            imageMedium: "static/alfalfa.jpg",
+            imageLarge: "static/alfalfa.jpg",
+            buyCost: 3,
+            sellCost: 1,
+            harvestYield: 4,
+            harvestTimeSeconds: 3
+        },
+        b: {
+            name: "Broccoli",
+            image: "static/broccoli.jpg",
+            imageSmall: "static/broccoli.jpg",
+            imageMedium: "static/broccoli.jpg",
+            imageLarge: "static/broccoli.jpg",
+            buyCost: 6,
+            sellCost: 1,
+            harvestYield: 9,
+            harvestTimeSeconds: 120
+        },
+        c: {
+            name: "Cabbage",
+            image: "static/cabbage.jpg",
+            imageSmall: "static/cabbage.jpg",
+            imageMedium: "static/cabbage.jpg",
+            imageLarge: "static/cabbage.jpg",
+            buyCost: 12,
+            sellCost: 2,
+            harvestYield: 10,
+            harvestTimeSeconds: 300
+        }
+    }
+};
+
+var state = {
+    cash: GAME_CONFIG.startingCash,
+    a: 0,
+    b: 0,
+    c: 0,
+//    for(i = 0, i < 2, i++) {
+//      for(j = 0, j < 2, j++) {
+//        state.append("plot" + i + j);
+//            state["plot" + i + j].append(seedType: 0, timeLeft: 0);
+//        }
+//    }
+    plot00: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot01: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot02: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot10: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot11: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot12: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot20: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot21: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot22: {
+        seedType: 0,
+        timeLeft: 0
+    }
+};
+
+buy = function(seed) {
+    if (state.cash >= GAME_CONFIG.seeds[seed].buyCost) {
```

very small point, but when doing verification checks, you'll keep your code more organized if you "fail early."  Your code would look like:
```
if (some error state) {
  alert on the error
  return;
}
if (some other error state) {
  alert on the other error
  return;
}
do your logic here for correct state
```
When you have tens of things to check, your indents will get very deep and linking up which alert goes with which check is tough.